bookwyrm/templates/rss/edition.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+{% blocktrans trimmed with book_title=obj.title book_author=obj.author_text %}
+  ‘{{ book_title }}’ by {{ book_author }}
+{% endblocktrans %}
+{{obj.description|default:""}}
```

Comment:
This shouldn't be a blocker because it seems to be a bit difficult to manage embedded HTML in the Django syndication framework, but I noticed this just runs on without line breaks so it renders like `Title of Book by Jo Bloggs Here is a description of the book`.

Reply:
I'm not sure what to do here. 

If I could figure out how to trigger a cdata block, we could do something to embed new lines. 
This seems like an approach someone got working, if it makes sense to you, I can take a swing at it.
https://nemecek.be/blog/33/how-to-create-rss-feed-with-html-content-in-django

For my purposes, I'm very interested in getting the book ISBN and other identifiers in as raw a format as possible, but I'd like to make sure this feed is useful and looks good for humans in a feed reader.

When I look at how other book tracking sites handle this, they use the cdata block pretty liberally. 
They also just introduce custom elements like isbn and asin directly in the feed. Which doesn't seem valid xml but it seems not to blow anything up.
I'm not ready to dedicate myself just yet to implementing a custom feed generator as described in https://docs.djangoproject.com/en/5.0/ref/contrib/syndication/#custom-feed-generators

Reply:
btw @hughrun - this is listed as a blocking change I think - assuming I'm not misreading githubs UI.

Reply:
I'll take a look at this shortly.